suppression-backup-service/cmd/main_test.go

```diff
@@ -53,7 +53,7 @@ func verifyBackup(t *testing.T, filePath, data string) {
 	require.NoError(t, err)
 	isSuppressed, err := repo.Suppressed("workspace-1", "user-1", "src-1")
 	require.NoError(t, err)
-	require.True(t, isSuppressed)
+	require.NotNil(t, isSuppressed)
```

Comment:
We need to be careful while rolling this out:
-Suppression backup service needs to be updated before gateways. 
- The service's pvc needs to be deleted as well so that no previous backup file is retained

Reply:
Another option would be to change paths for the full backup & export files so that the service doesn't pickup the old ones.

Reply:
Made change in this commit: https://github.com/rudderlabs/rudder-server/pull/3542/commits/de8d3e3538284164ba15eeca75c80140209548f0

Reply:
Changing the path for exported files is one aspect, changing the path for badgerDB is another :) 

BadgerDB for `fullsuppression` survives restarts, whereas badgerDB for `latestsuppression` doesn't.